helmfile/charts/prometheus-alerts/templates/alerts/kubernetes-apps.yaml

```diff
@@ -260,5 +260,16 @@ spec:
         severity: warning
 {{- if .Values.defaultRules.additionalRuleLabels }}
 {{ toYaml .Values.defaultRules.additionalRuleLabels | indent 8 }}
+{{- end }}
+    - alert: KubePodEvicted
+      annotations:
+        description: '{{`{{`}} $value {{`}}`}} pods were Evicted from {{`{{`}} $labels.namespace {{`}}`}} namespace {{`{{`}} $labels.cluster {{`}}`}} cluster in the last 10 minutes.'
+        summary: Kubernetes pods got evicted.
+      expr: sum by (namespace, cluster) (kube_pod_status_reason{reason="Evicted"}) >= 1
+      for: 10m
+      labels:
+        severity: warning
+{{- if .Values.defaultRules.additionalRuleLabels }}
+{{ toYaml .Values.defaultRules.additionalRuleLabels | indent 8 }}
```

Comment:
Isn't this a duplicate? Looks like these are present at line 261-262

Reply:
this lines are added for each alert, to add additional labels, if used
I think they are from the upstream files
do you want to remove them?